kernel/lib/time.hpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file time.hpp
+ * @author Michel (JMallone) Gomes (michels@utfpr.edu.br)
+ * @brief Time lib code 
+ * @version 0.1
+ * @date 2021-07-27
+ *
+ * @copyright Copyright the Panix Contributors (c) 2021
+ *
+ * References:
+ *              https://github.com/sidsingh78/EPOCH-to-time-date-converter/blob/master/epoch_conv.c
+ *              https://www.oryx-embedded.com/doc/date__time_8c_source.html
+ */
+#pragma once
+
+namespace Time {
+
+class TimeDescriptor {
+public:
+    // Constructors
+    TimeDescriptor(int seconds, int minutes, int hours, int days, int month, int year);
+    TimeDescriptor();
+    ~TimeDescriptor();
+    // Getters
+    int getSeconds() { return _seconds; }
+    int getMinutes() { return _minutes; }
+    int getHour() { return _hour; }
+    int getDay() { return _day; }
+    int getMonth() { return _month; }
+    int getYear() { return _year; }
+    // Setters
+    void setSeconds(int newSeconds) { _seconds = newSeconds; }
```

Comment:
I know this is super nit-picky, but I would prefer the parameters be something like `sec`, `min`, `hr`, `d`, `m`, `y` instead of `newSeconds`, etc.